books/free-programming-books-subjects.md

```diff
@@ -252,6 +252,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Dotnet Microservices Architecture for Containerized NET Applications](https://aka.ms/microservicesebook) - Cesar de la Torre, Bill Wagner, Mike Rousos (PDF)
 * [Kubernetes Deployment & Security Patterns](https://resources.linuxfoundation.org/LF+Projects/CNCF/TheNewStack_Book2_KubernetesDeploymentAndSecurityPatterns.pdf) - Alex Williams (PDF)
 * [Kubernetes for Full-Stack Developers](https://www.digitalocean.com/community/books/digitalocean-ebook-kubernetes-for-full-stack-developers) - Jamon Camisso, Hanif Jetha, Katherine Juell (PDF, EPUB)
+* [Kubernetes Ingress Controller Deployment and Security with NGINX](https://www.nginx.com/resources/library/kubernetes-ingress-controller-deployment-security-nginx/#download) - Amir Rawdat, Akash Ananthanarayanan
```

Comment:
We prioritize freely accessible resources.

We won't include sites requiring email for book access, as per our contribution guidelines. 

Also, we avoid listing books that need a working email address for access.